website/docs/r/nsxt_distributed_firewall_rule.html.markdown

```diff
@@ -0,0 +1,122 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_distributed_firewall_rule"
+sidebar_current: "docs-vcd-resource-nsxt-distributed-firewall-rule"
+description: |-
+  The Distributed Firewall allows user to segment organization virtual data center entities, such as
```

Comment:
This description is adequate for the whole set of rules, but feels out of sort for a single rule.

Reply:
Fixed.